source/write-scripts.txt

```diff
@@ -157,20 +157,21 @@ You can ``require()`` native Node modules (such as
 
       var fs = require('fs');
 
-Require a Non-Native Module
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Require a Non-Built-In Module
```

Comment:
Would something like "External" be preffered over "Non-Built-In" ? Or perhaps some other name?

Reply:
Yes, I think so, but also you're referencing npm packages here explicitly, so I think just saying "npm modules" is fine 🙂